### PR TITLE
239 cdi20 initial drop

### LINKED ISSUE #239: CDI 2.0: reorganise all weld integration
Reorganise all weld integration bundles so that both CDI 1.2 and CDI 2.0 can share the same integration classes

### PR DESCRIPTION
**Pull request by tevans78**
fixes #239
#build

No way this will pass the feature checker just yet!

### CODE REVIEW

**Comment by tevans78** [comment]
Added a WL tweak to temporarily disable the CDI 1.2 FAT bucket. A build of the combined WL and OL changes is here:
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_w-AVEKz_Eee9vu121oMBnw

And an IM build here:
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_6l6qgKz_Eee9vu121oMBnw

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.websphere.appserver.cdi-2.0/resources/l10n/com.ibm.websphere.appserver.cdi-2.0.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2014 IBM Corporation and others.
```
Comment (Emily-Jiang) [inline dev/com.ibm.websphere.appserver.cdi-2.0/resources/l10n/com.ibm.websphere.appserver.cdi-2.0.properties]: Wrong copyright year

File: dev/com.ibm.websphere.appserver.cdi1.2-jndi1.0/resources/com.ibm.websphere.appserver.cdi1.2-jndi1.0.mf [hunk 2]
```diff
@@ -7,7 +7,7 @@ IBM-License-Agreement: wlp/lafiles/LA
 IBM-ProductID: com.ibm.websphere.appserver
 IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.cdi-1.2))",
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jndi-1.0))"
-Subsystem-Content: com.ibm.ws.cdi.1.2.jndi; version="[1.0.0,1.0.200)"
+Subsystem-Content: com.ibm.ws.cdi.jndi; version="[1.0.0,1.0.200)"
```
Comment (Emily-Jiang) [inline dev/com.ibm.websphere.appserver.cdi1.2-jndi1.0/resources/com.ibm.websphere.appserver.cdi1.2-jndi1.0.mf]: Is this file needed?

Comment (Emily-Jiang) [reply dev/com.ibm.websphere.appserver.cdi1.2-jndi1.0/resources/com.ibm.websphere.appserver.cdi1.2-jndi1.0.mf]: Talked to Erik, Andy. Glenn Marcy is going to delete all of the .mf files.
